src/main/java/manufacturing/dao/jdbc/ManufacturerDaoJdbcImpl.java

```diff
@@ -0,0 +1,125 @@
+                + "WHERE manufacturer_id = ? AND `delete` = FALSE";
+        Manufacturer manufacturer = null;
+        try (Connection connection = ConnectionUtil.getConnection();
+            PreparedStatement preparedStatement = connection.prepareStatement(query)) {
+            preparedStatement.setLong(1, manufacturerId);
+            ResultSet resultSet = preparedStatement.executeQuery();
+            while (resultSet.next()) {
+                manufacturer = createManufacturer(resultSet);
+            }
+            return Optional.ofNullable(manufacturer);
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get manufacturer by id "
+                    + manufacturerId, e);
+        }
+    }
+
+    @Override
+    public Manufacturer update(Manufacturer manufacturer) {
+        String query = "UPDATE manufacturers "
+                + "SET manufacturer_name = ?, manufacturer_country = ?"
+                + "WHERE manufacturer_id = ? AND `delete` = FALSE";
+        try (Connection connection = ConnectionUtil.getConnection();
+            PreparedStatement preparedStatement = connection.prepareStatement(query)) {
+            preparedStatement.setString(1, manufacturer.getName());
+            preparedStatement.setString(2, manufacturer.getCountry());
+            preparedStatement.setLong(3, manufacturer.getId());
+            preparedStatement.executeUpdate();
+            return manufacturer;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't update manufacturer in BD", e);
+        }
+    }
+
+    @Override
+    public boolean delete(Long manufacturerId) {
+        String query = "UPDATE manufacturers "
+                + "SET `delete` = TRUE "
+                + "WHERE manufacturer_id = ?";
+        int deletedId;
+        try (Connection connection = ConnectionUtil.getConnection();
+            PreparedStatement preparedStatement = connection.prepareStatement(query)) {
+            preparedStatement.setLong(1, manufacturerId);
+            deletedId = preparedStatement.executeUpdate();
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't delete manufacturer by following id "
+                    + manufacturerId, e);
+        }
+        return deletedId > 0;
+    }
+
+    @Override
+    public List<Manufacturer> getAllManufacturers() {
+        String query = "SELECT * FROM manufacturers "
+                + "WHERE `delete` = FALSE";
+        List<Manufacturer> manufacturers = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+            PreparedStatement preparedStatement = connection.prepareStatement(query)) {
+            ResultSet resultSet = preparedStatement.executeQuery();
+            while (resultSet.next()) {
+                manufacturers.add(createManufacturer(resultSet));
+            }
+            preparedStatement.close();
+            return manufacturers;
```

Comment:
```suggestion
            }
            return manufacturers;
```
statement is inside of try with resources, we don't need to close it explicitly in try

Reply:
sorry. my mistake. forgot to remove it after changing